packages/block-editor/src/components/block-list/style.scss

```diff
@@ -501,6 +501,11 @@
  * Block Toolbar when contextual.
  */
 
+// Show a small gap above the block toolbar when it stickies to the top of the screen.
+.block-editor-block-contextual-toolbar-wrapper {
+	padding-top: $grid-unit-15;
```

Comment:
Does this interfere at all with clicking on the previous block, etc?

Reply:
Good thought. Yes, it looks like it does.

<img width="753" alt="Screenshot 2021-04-20 at 12 28 17" src="https://user-images.githubusercontent.com/1204802/115381476-e5c50d80-a1d3-11eb-9797-4a8507c52396.png">

Let me look at alternatives.

Reply:
Looks like there aren't any good CSS solutions to this one, and that we'll need to add that gap in the positioning math. CC: @ellatrix

Reply:
I see you found pointer-events :)